nodes/config/ui_base.html

```diff
@@ -239,16 +239,22 @@
         min-height: 20px; /* IMPORTANT! So that user when last element is dragged out it doesnt collapse. */
     }
     div.nrdb2-sb-widget-list-container ol.nrdb2-sb-widget-list.red-ui-editableList-list.ui-sortable:empty {
-        border: 2px dashed lightgrey;
+        border: 1px dashed #c4c4c4;
         padding: 7px; /* should be 9px to match the other entries but border width is 2px */
```

Comment:
@joepavitt to avoid things jumping around you might want to increase the padding so that the overall hight matches the native "empty" placeholder 

No biggie tho!

Reply:
Good spot - consider it done